testing/conformance/Cargo.toml

```diff
@@ -44,7 +44,7 @@ serde_json = { version = "1.0", features = ["raw_value"] }
 walkdir = "2.3"
 regex = { version = "1.0" }
 ittapi-rs = { version = "0.1.6", optional = true }
-actors-v6 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "v6" }
+actors-v6 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "release/v6" }
```

Comment:
Did you create this branch?

Reply:
Yes i did

Reply:
This is the branch name we converged to for long-term maintenance of release lines.